index.js

```diff
@@ -59,7 +65,13 @@ module.exports = (input, opts) => {
 				task: () => del('node_modules')
 			},
 			{
-				title: 'Installing dependencies',
+				title: 'Installing dependencies using Yarn',
+				enabled: () => opts.yarn === true,
+				task: () => exec('yarn', ['install'])
```

Comment:
Oh okay, didn't event know it worked :p. I always type `yarn` because it's faster. So fine for me for using the more verbose one.